Files/Interacts/Interaction.cs

```diff
@@ -939,6 +900,7 @@ public async void PasteItem_ClickAsync(object sender, RoutedEventArgs e)
                     if (destinationPath.Contains(item.Path, StringComparison.OrdinalIgnoreCase))
                     {
                         ImpossibleActionResponseTypes responseType = ImpossibleActionResponseTypes.Abort;
+                        var resourceLoader = Windows.ApplicationModel.Resources.ResourceLoader.GetForViewIndependentUse();
                         ContentDialog dialog = new ContentDialog()
                         {
                             Title = resourceLoader.GetString("ErrorDialogThisActionCannotBeDone"),
```

Comment:
What about to use [REsourceController](https://github.com/duke7553/files-uwp/blob/develop/Files/ResourceController.cs) instead of when you need resources any time call ResourceLoader?

Reply:
This is code carried over from https://github.com/duke7553/files-uwp/commit/0ace5ecc98cab6b5c0fdc4d80c7c7ba7ee078dfc. I can change it to use `ResourceLoader .GetTranslation("ErrorDialogThisActionCannotBeDone")` instead.

Reply:
Moreover, calling ResourceLoader in foreach loop possibly could degrade performance

Reply:
Changed in https://github.com/duke7553/files-uwp/pull/586/commits/8cac43de0c71b862a548fcb627c84d53f289a709.